cli/ballerina-cli/src/main/resources/cli-help/ballerina-shell.help

```diff
@@ -0,0 +1,33 @@
+NAME
+       ballerina-shell - Run ballerina interactive REPL
+
+SYNOPSIS
+       bal shell [-d|--debug] [-f|--force-dumb]  [-t|--time-out]
+
+
+DESCRIPTION
+       Shell runs a REPL instance of ballerina to enable users to run small
+       snippets to code to check their outputs. Please not that this is an
```

Comment:
```suggestion
       snippets of code and check the output. Please note that this is an
```

Do we need "check the output" part at all?

Reply:
I'll remove the "check the output" part